### PR TITLE
4985: create new user pool for irs superuser

### PR DESCRIPTION
**Pull request by rachaelparris**

### CODE REVIEW

**Review by codyseibert — APPROVED** [review]

File: web-api/terraform/template/cognito.tf [hunk 1]
```diff
@@ -100,3 +100,112 @@ resource "aws_cognito_user_pool_domain" "main" {
...
+  lambda_config {
+    post_confirmation = "${aws_lambda_function.cognito_post_confirmation_lambda.arn}"
+  }
+
+  admin_create_user_config {
+    allow_admin_create_user_only = true
+    invite_message_template {
+      sms_message = "Your username is {username} and temporary password is {####}."
+      email_subject = "U.S. Tax Court account creation"
+      email_message = "An account has been created for you on the <a href='https://ui-dev.ustc-case-mgmt.flexion.us/'>U.S. Tax Court site</a>. Your username is {username} and temporary password is {####}. Please log in and change your password."
```
Comment (matthopson) [inline web-api/terraform/template/cognito.tf:143]: We should remember to move the password out of the sentence - maybe put it on a new line?

**Review by matthopson — APPROVED** [review]

File: web-api/terraform/template/cognito.tf (hunk 1 above)
Comment (rachaelparris) [reply web-api/terraform/template/cognito.tf:143]: I think I'll wait for UX to give us some verbiage for this so it's right.

**Comment by codyseibert** [comment]
could you figure out how to enabled the remember device settings and max device one?

File: web-api/terraform/template/cognito.tf (hunk 1 above)
Comment (matthopson) [reply web-api/terraform/template/cognito.tf:143]: 👍 Is there a task noted somewhere for UX for this?

Comment (rachaelparris) [reply web-api/terraform/template/cognito.tf:143]: Yep, here's the trello card: https://trello.com/c/VB4dZVer/483-rewrite-reformat-new-user-email-messaging

Comment (matthopson) [reply web-api/terraform/template/cognito.tf:143]: Tanks

Comment (kkoskelin) [reply web-api/terraform/template/cognito.tf:143]: I know I'm not a UXer but my thought was that getting some newlines in here would be good.  e.g.
```
Your username is {placeholder}
Your password is {placeholder}
```

Also, perhaps the period should be removed from the SMS mesage.

**Pull request closed without merging**